### ISSUE TITLE
MissingJobTimeout false positive: ${{ }}

### ISSUE THREAD

**Issue by TWiStErRob** [TWiStErRob/net.twisterrob.ghlint#107]
```
jobs:
  integration-test:
    name: "Integration Tests"
    runs-on: ubuntu-latest
    timeout-minutes: ${{ inputs.timeout-minutes }}
```

```
MissingJobTimeout at act-integration-test.yml:45:3-19: Job[integration-test] is missing `timeout-minutes`.
```

Version: 0.3.1
